api/v1alpha1/webhooks/releaseplan/webhook.go

```diff
@@ -46,6 +47,10 @@ func (w *Webhook) Default(ctx context.Context, obj runtime.Object) error {
 		}
 	}
 
+	if releasePlan.Spec.ReleaseGracePeriodDays == 0 {
```

Comment:
So if you omit the field, it equals 0? If so, that is a little unfortunate... it would be nice if setting it to 0 was a real possibility when a user wants their Release CRs to never expire

Reply:
Yes. I initially expected it would be null, but it is indeed zero.

Reply:
That's always the case for `int`. I know other team "fixed" this by using `*int`, but I don't really like that solution.